docs/static/releasenotes.asciidoc

```diff
@@ -3,14 +3,32 @@
 
 This section summarizes the changes in the following releases:
 
-* <<logstash-5-6-6,Logstash 5.6.6>>
+* <<logstash-5-6-7,Logstash 5.6.6>>
 * <<logstash-5-6-5,Logstash 5.6.5>>
 * <<logstash-5-6-4,Logstash 5.6.4>>
 * <<logstash-5-6-3,Logstash 5.6.3>>
 * <<logstash-5-6-2,Logstash 5.6.2>>
 * <<logstash-5-6-1,Logstash 5.6.1>>
 * <<logstash-5-6-0,Logstash 5.6.0>>
 
+[[logstash-5-6-7]]
+=== Logstash 5.6.7 Release Notes
+
+*Gelf Input*
```

Comment:
I think you need a blank line here for the markdown.

Reply:
Done